tools/api-markdown-documenter/api-report/api-markdown-documenter.api.md

```diff
@@ -320,7 +324,19 @@ export enum ListKind {
 }
 
 // @public
-export function loadModel(reportsDirectoryPath: string): Promise<ApiModel>;
+export function loadModel(reportsDirectoryPath: string, logger?: Logger): Promise<ApiModel>;
+
+// @public
+export interface Logger {
+    error: LoggingFunction;
+    log: LoggingFunction;
```

Comment:
nit: I like `info` for the base logging level, because then you can name the logger `log` and use `log.info(...)`, `log.verbose(...)` etc.